janitor/functions.py

```diff
@@ -29,6 +57,7 @@ def clean_names(df):
         df = jn.DataFrame(df).clean_names()
 
     :param df: The pandas DataFrame object.
+    :param strip_underscores: A str of either 'left', 'right' or 'both'.
```

Comment:
Likewise here, please document the options for clarity purposes, and what they're intended to do.